public/index.html

```diff
@@ -19,9 +19,20 @@
                         <th>Name</th>
                         <th>Email</th>
                         <th>Number</th>
+                        <th>Action</th>
                     </tr>
                 </thead>
                 <tbody>
+                    <tr>
+                        <td><input ng-model="contact.name" class="form-control"/></td>
```

Comment:
These input fields are currently using the `$scope` syntax. As stated before, I want to change this to use a controller in the next ticket.